### PR TITLE
Run tests in parallel

### PR DESCRIPTION
**Pull request by thewilkybarkid**

### CODE REVIEW

File: tests/SchemaTest.php [hunk 1]
```diff
@@ -65,13 +65,13 @@ public function unsupported_elements_are_flagged(Document $input, XmlValidator $
         }
     }
 
-    public function fileProvider() : iterable
+    public function fileProvider() : array
```
Comment (thewilkybarkid) [inline tests/SchemaTest.php:70]: Paratest seems to require an `array`.

Comment (thewilkybarkid) [reply tests/SchemaTest.php:70]: https://github.com/paratestphp/paratest/pull/398

**Comment by thewilkybarkid** [comment]
Hmm, not sure can make a difference in Travis.

**Comment by giorgiosironi** [comment]
Can confirm works locally cutting 80 seconds to 40 seconds. Tests are likely CPU-bound and saturating the (single?) core available on Travis CI VMs? Could run some diagnostic commands to find out what the environment provides.

An approach like using [multiple jobs](https://docs.travis-ci.com/user/speeding-up-the-build/#parallelizing-your-builds-across-virtual-machines) may be more promising since we don't have to collect a unique result; assumes there is a clear way to split tests in similarly sized chunks.

**Pull request closed without merging**